js/event-domcontentloaded/demos/overloaded/index.html

```diff
@@ -15,7 +15,11 @@
     }
 
```

Comment:
Тут скролы есть, их бы покрасить

```suggestion

    html {
      color-scheme: dark;
    }

```